index.html

```diff
@@ -0,0 +1,186 @@
+				</div>
+				<button type="button" class="project_btn btn">See project</button>
+			</div>
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button type="button" class="project_btn btn">See project</button>
+			</div>
+			
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button type="button"  class="project_btn btn">See project</button>
+			</div>
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button  type="button" class="project_btn btn">See project</button>
+			</div>
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button" >HTML</button>
+					<button type="button" >Boostrap</button>
+					<button type="button" >Ruby</button>
+				</div>
+				<button type="button" class="project_btn btn">See project</button>
+			</div>
+		</div>
+	</section>
+	<section class="about_container">
+		<section class="about">
+			<h1>About me</h1>
+		<p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+		<button type="button" >Get my resume</button>
+		</section>
+		<section class="">
+		<p class="linep"></p>
+		<ul class="list">
+			
+			<li class="tit"><p class="p"></p><h3>Language</h3></li>
+			<li>Javascript</li>
+			<li>Ruby</li>
+			<li>HTML</li>
+			<li>CSS</li>
+		</ul>
+		<ul class="list">
+			
+			<li class="tit"><p class="pp"></p><h3>Frameworks</h3></li>
+			<li>React.js</li>
+			<li>Ruby on Rails</li>
+			<li>RSpec</li>
+			<li>Capybara</li>
+		</ul>
+		<ul class="list">
+			
+			<li class="tit"><p class="ppp "></p><h3>Skills</h3></li>
+			<li>Database Management</li>
+			<li>Version Control</li>
+			<li>CLI</li>
+			<li>API Design</li>
```

Comment:
Good job using the correct layout method👍 

Please let's consider changing the `font` and use the same `font` as the template. 👍💪

Your design
![image](https://user-images.githubusercontent.com/78251927/229277947-5c0c171c-9d37-4a9d-bab5-b66f0a57b930.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/229277970-be2fbb80-3c40-4441-9916-eacdeaa9f12b.png)